com.unity.ml-agents/Runtime/Timer.cs

```diff
@@ -376,14 +384,26 @@ public string DebugGetTimerString()
         /// <param name="filename"></param>
         public void SaveJsonTimers(string filename = null)
         {
-            if (filename == null)
+            try
+            {
+                if (filename == null)
+                {
+                    var activeScene = SceneManager.GetActiveScene();
```

Comment:
I can be talked out of this.

Reply:
Is the SaveJsonTimers always first called after at least one step happens? In a lot of games the initial scene isn't the one containing the agents (e.g. title scene, etc.).

Reply:
It's only called on shutdown. SceneManager.GetActiveScene() returns a struct so there's no danger of it being `null`. It's not _that_ important that we get the scene name, I just thought it would be more useful than the current approach of the hard-coded name (`csharp_timers.json`)